install/kubernetes/cilium/templates/cilium-agent/validatingadmissionpolicybinding.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if and .Values.agent (not .Values.preflight.enabled) }}
+---
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: "http-tcp-binding"
+spec:
+  policyName: "http-tcp"
+  matchResources:
```

Comment:
I have not yet looked at the validation feature in detail. Will this empty match statement all resources?

Reply:
> I have not yet looked at the validation feature in detail. Will this empty match statement all resources?

Yep. I used an empty LabelSelector for "matchResources", which will match all objects